app/models/institution_category_ratings_archive.rb

```diff
@@ -0,0 +1,4 @@
+# frozen_string_literal: true
+
+class InstitutionCategoryRatingsArchive < ApplicationRecord
```

Comment:
For consistency reasons, wondering if this should be InstitutionCategoryRatingArchive instead of InstitutionCategoryRatingsArchive and the corresponding table renamed as well.